src/elements/ExpiryIndicator.stories.ts

```diff
@@ -0,0 +1,45 @@
+import {fn} from '@storybook/test';
+import type {Meta, StoryObj} from '@storybook/vue3';
+
+import BubbleSelect from "@/elements/BubbleSelect.vue";
+import type {SelectOption} from "@/models";
```

Comment:
Unused. Can we remove those?

Reply:
I saw these unused in other files too so I'm not entirely sure.

Reply:
I see, alright. I think just let them be for now and we will do a proper clean up afterwards. Thank you!